wiki/osu!_wiki/Maintenance/en.md

```diff
@@ -0,0 +1,103 @@
+
+## Technical details
+
+### Issue tracker
+
+The osu! wiki's [issue tracker](https://github.com/ppy/osu-wiki/issues) contains ideas and requests for possible improvements, both for the articles themselves and wiki-related parts of the website. Open an issue if you have a feature request, or if you found a mistake on one of the pages. Please note that **this is only limited to the osu! wiki**—if you need assistance with other official projects related to osu!, use their own issue trackers:
+
+- [osu!(lazer)](https://github.com/ppy/osu)
+- [The osu! website](https://github.com/ppy/osu-web/)
+- [osu!(stable) issues](https://github.com/ppy/osu-stable-issues)
+
+#### Issue labels
+
+On GitHub, pull requests and issues may be tagged and classified by using [labels](https://github.com/ppy/osu-wiki/labels), which show different aspects of a pull request or issue. Labels are informational, set by the wiki maintainers, and are typically self-explanatory. While they require no action from a user's perspective, red labels serve as reminders or call to action for other maintainers:
+
+- `rule change`: the change affects an existing set of rules, such as the [Ranking Criteria](/wiki/Ranking_Criteria), and needs to be reviewed by the area's owner
+- `blocked`: the change has issues which must be resolved before proceeding, or depends on another issue which must be resolved first
+- `needs native review`: the translation needs to be checked by a person fluent in the respective language. Alternatively, during the merge process it means that no such review took place
+- `needs rebase`: the pull request has too many small unstructured commits, which need to be rebased and worded in a better way
+
+### Links and redirects
+
+Most osu! wiki articles have alternative links, which are set up using the [`redirect.yaml`](https://github.com/ppy/osu-wiki/blob/master/wiki/redirect.yaml) file. The redirects are meant to be used outside the osu! wiki, for example, on forums, or in the [chat](/wiki/Chat_Console), where they could be quickly turned into an inline reference:
+
+```
+According to [[RC]], this is forbidden.
+```
+
+When adding redirects for a new or existing article, keep in mind that they should be concise and designed for actual use.
+
+<!-- TODO: should probably briefly mention external links https://github.com/ppy/osu-web/issues/8086 and footnotes https://github.com/ppy/osu-wiki/issues/4911#issuecomment-893959588 once they are implemented -->
+
+### CI checks
+
+The osu! wiki repository uses [continuous integration](https://docs.github.com/en/actions/guides/about-continuous-integration) (CI) to automatically check incoming pull requests for different common errors. The list of checks is configured in the [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) file and includes several custom plugins written by the osu! wiki maintainers.
```

Comment:
```suggestion
The osu! wiki repository uses [continuous integration](https://docs.github.com/en/actions/guides/about-continuous-integration) (CI) to automatically check incoming pull requests for different common errors. The list of checks is configured in the [`continuous-integration.yml`](https://github.com/ppy/osu-wiki/blob/master/.github/workflows/continuous-integration.yml) file. 

The [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) file lists all plugins used by the CI, from which some were written by the osu! wiki maintainers.
```

Reply:
I was referring to the checks themselves (= remark plugins), not CI workflows, but I like your option more